Gemfile

```diff
@@ -1,97 +1,97 @@
 # A sample Gemfile
-source "https://rubygems.org"
+source 'https://rubygems.org'
 
-gem "rails", '~>6.0.3'
-gem "pg", '~> 0.20.0'
 gem 'activerecord-import'
+gem 'pg', '~> 0.20.0'
+gem 'rails', '~>6.0.3'
 
+gem 'bcrypt'
+gem 'bootsnap'
 gem 'composite_primary_keys', '~> 12.0'
+gem 'csv', '>= 1.0.2' # support for bom|utf-8 in ruby 2.5
 gem 'order_as_specified'
-gem 'bootsnap'
-gem 'bcrypt'
 gem 'with_advisory_lock'
-gem 'csv', '>= 1.0.2' # support for bom|utf-8 in ruby 2.5
 
-gem "haml-rails"
 gem 'autoprefixer-rails'
-#updating this to 4.5.3 causes a weird missing variable bug
+gem 'haml-rails'
+# updating this to 4.5.3 causes a weird missing variable bug
```

Comment:
I think this comment goes with the bootstrap line?

Reply:
Ugh, yes, Rubocop wanted them all sorted, I didn't catch that it's not smart enough to move the associated comments.